### ISSUE TITLE
Saving of template doesn't work outside of notebook

### ISSUE THREAD

**Issue by cpsievert** [trungleduc/ipyflex#9]
Thanks for this great package! I noticed that `FlexLayout()` can be rendered statically, but in that case, the template saving doesn't work (it'd be awesome if it did) 

```python
from ipyflex import FlexLayout
import ipywidgets as ipw

widgets = {
    "Widget 1": ipw.HTML("<h1>Widget 1</h1>"),
    "Widget 2": ipw.HTML("<h1>Widget 2</h1>"),
    "Widget 3": ipw.HTML("<h1>Widget 3</h1>"),
    "Widget 4": ipw.HTML("<h1>Widget 4</h1>"),
}
w = FlexLayout(widgets, style={"height": "400px"})

from ipywidgets.embed import embed_minimal_html
embed_minimal_html("index.html", views=[w])
```

**Comment by trungleduc** [comment]
Indeed it does not work since the save function is processed in the backend. But I think I can add an option to download and upload the template from the interface without using the backend.  In this case, you can configure your static dashboard at run time.

**Comment by trungleduc** [comment]
Resolved with #21
